README.md

```diff
@@ -1,2 +1,41 @@
-# hw01
-Repository for homework 1
+# HW01
+## Katherine M. Surma
+### About Me
+
+I am a joint-degree BA/MA student at the University of Chicago. I am earning my BA in Political Science and Law, Letters, & Society, and I am completing my Master's degree through the [Master of Arts Program in the Social Sciences](https://mapss.uchicago.edu/node).  
+
+![I'm on the right in the above picture.](katherine.jpg)
+
+### Research Interests
+
+* American politics
```

Comment:
The lists look great in both the the code and its rendering! Also the picture is something I couldn't figure out. I had overlooked the required (!)